### ISSUE TITLE
MACHINA_DEFAULT_AUTHENTICATED_USER_FORUM_PERMISSIONS can't get to work

### ISSUE THREAD

**Issue by pspierce** [ellmetha/django-machina#58]
I have about 30 users and 8 groups.  The groups are departments within the company.  Each department has several default forums which they only have permissions to view, create, edit... etc.  I'd like to create a number of forums that all users/departments can access.  I thought I could do this with the default permissions within settings.  If I read correctly, that is exactly why the setting exists; however, when creating a new forum and not setting any permissions, users still can't see the forum despite the default permissions allowing it.

**Comment by ellmetha** [comment]
Is it a top-level forum? A forum inside a category? The permissions defined in `MACHINA_DEFAULT_AUTHENTICATED_USER_FORUM_PERMISSIONS` are used only if the forum does not have any other permissions. Anyway, could you provide some simple steps allowing me to reproduce the problem?

**Comment by pspierce** [comment]
I have a top-level forum:
```
In [19]: f.get_type_display()
Out[19]: u'Category forum'
In [20]: f.userforumpermission_set.all()
Out[20]: <QuerySet []>
In [21]: f.groupforumpermission_set.all()
Out[21]: <QuerySet []>
In [23]: settings.MACHINA_DEFAULT_AUTHENTICATED_USER_FORUM_PERMISSIONS
Out[23]: 
[u'can_see_forum',
 u'can_read_forum',
 u'can_start_new_topics',
 u'can_reply_to_topics',
 u'can_edit_own_posts',
 u'can_post_without_approval',
 u'can_create_polls',
 u'can_vote_in_polls',
 u'can_download_file']

```

With all of the above, the forum still can't be seen be an authenticated user.

**Comment by BrendaH** [comment]
Why was this issue closed without a solution?

I also have this issue.
I have 2 category-toplevel forums with specific permissions for specific groups. Now I want to add another category-toplevel forum that everybody who is logged in can see, so I created that forum with a subforum and one topic and made sure there were no permissions set. But with the setting MACHINA_DEFAULT_AUTHENTICATED_USER_FORUM_PERMISSIONS with read, write and download attachment options a logged in user still can't see the newly made permission-free forum. 

The documentation states "In that case, this setting can be used in order to define which permissions should be granted to all authenticated users. Note that the permissions specified in this list are granted only if the considered forum does not have any permission for the considered authenticated user. " Which is exactly what I set up.

I do not want to set rights for all current groups on this new category-toplevel forum because in the future new groups will be made and that is not always done by me. So I can not always add that group to the permission list for this new forum. Therefore I want to grant all authenticated users the right, by default, to see this forum and it's contents.

What am I missing?
